src/app/components/ConnectorForm/index.tsx

```diff
@@ -56,7 +56,7 @@ function ConnectorForm({
 
   return (
     <form onSubmit={onSubmit}>
-      <div className="max-w-xl mx-auto mt-6 relative bg-white dark:bg-surface-02dp p-7 shadow rounded-lg">
+      <div className="max-w-xl mx-auto mt-6 relative bg-white dark:bg-transparent dark:bg-surface-02dp p-7 shadow rounded-lg">
```

Comment:
there is a duplicated bg class here, can you remove it?

Reply:
Sure I will remove it.

Reply:
removed it!